app/controllers/helper/AuthControllerHelpers.scala

```diff
@@ -24,14 +24,15 @@ import play.api.mvc.Results._
 import scala.concurrent.Future
 
 trait AuthControllerHelpers extends Authenticated with Logging {
-  def authenticationResultHandler(methodName: String)(authResult: AuthenticationResult): Future[Result] =
+  def authenticationFailureResultHandler(methodName: String)(authResult: AuthenticationResult): Future[Result] =
     authResult match {
       case NotLoggedIn =>
         logger.info(s"[Authentication] [$methodName] User not logged in")
         Future.successful(Forbidden)
+      case _ => Future.successful(Forbidden)
```

Comment:
Is the extra case here necessary? Shouldn't line 29 just be the catch all case with the forbidden messages?